dash/_hooks.py

```diff
@@ -0,0 +1,93 @@
+import typing as _t
+
+_ns = {
+    "setup": [],
+    "layout": [],
+    "routes": [],
+    "error": [],
+    "callback": [],
+}
+
+
+def layout(func):
+    """
+    Run a function when serving the layout, the return value
```

Comment:
This looks like it saves the function `func` but doesn't run it - is the comment incorrect?

Reply:
No, it is only run `when` serving the layout, the docstring are placeholder, we can improve the wording.